en/docs/insights/generate-custom-reports.md

```diff
@@ -0,0 +1,25 @@
+# Generate Custom Reports
+
+Choreo insights allows you to generate custom reports to view the information that is important to you to make timely decisions for the betterment of your business. With custom reports, you can generate reports for a set of metrics of your choice and aggregate results by specific fields. Once generated, the metrics can be filtered out using any selected group-by field(s). Custom reports supports three visualizations: overtime charts, pie charts, and tables.
```

Comment:
Choreo insights allow you to generate custom reports to view the information that is important to you to make timely decisions for the betterment of your business. With custom reports, you can generate reports for a set of metrics of your choice and aggregate results by specific fields. Once generated, the metrics can be filtered out using any selected group-by filters. Custom reports support three visualizations: overtime charts, pie charts, and tables.